src/studio/src/designer/backend/gulpfile.js

```diff
@@ -346,10 +348,52 @@ gulp.task(
   ]),
 );
 
-gulp.task('fix-for-local-development', function () {
-  gulp
-    .src(['./Views/ServiceDevelopment/index.cshtml'])
-    .pipe(replace('~/designer/js/react', 'http://localhost:8080'))
-    .pipe(replace('~/designer/css/react', 'http://localhost:8080'))
-    .pipe(gulp.dest('./Views/ServiceDevelopment'));
-});
+gulp.task(
+  'local-dev-fix',
+  gulp.series(
+    function () {
+      return gulp
+        .src(['./Views/ServiceDevelopment/index.cshtml'])
+        .pipe(replace('~/designer/js/react', 'http://localhost:8080'))
+        .pipe(replace('~/designer/css/react', 'http://localhost:8080'))
+        .pipe(gulp.dest('./Views/ServiceDevelopment'));
+    },
+    function () {
+      return gulp
+        .src(['./Views/Home/index.cshtml'])
+        .pipe(replace('~/designer/js/react', 'http://localhost:8082'))
+        .pipe(replace('~/designer/css/react', 'http://localhost:8082'))
+        .pipe(gulp.dest('./Views/Home'));
+    },
+    function (cb) {
+      const { status } = spawnSync(
+        'git update-index --assume-unchanged Views/Home/Index.cshtml',
+      );
+      cb(status);
+    },
+    function (cb) {
+      const { status } = spawnSync(
+        'git update-index --assume-unchanged Views/ServiceDevelopment/index.cshtml',
+      );
+      cb(status);
+    },
+  ),
+);
+
+gulp.task(
+  'local-dev-rollback',
+  gulp.series([
+    function (cb) {
+      const { status } = spawnSync(
+        'git checkout origin/master -- Views/Home/Index.cshtml',
+      );
+      cb(status);
+    },
+    function (cb) {
+      const { status } = spawnSync(
+        'git checkout origin/master -- Views/ServiceDevelopment/index.cshtml',
```

Comment:
These trigger security warnings in Sonar...
They can be changed to `/usr/bin/git ...`

Reply:
I'll leave this here for now. This code was added for convenience as there is no way swap between dev-mode and production mode. So this is temporary code untill some backend developer creates a smarter way of doing this, or we change arcitecture.